zapsentry/example/main.go

```diff
@@ -5,9 +5,9 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/getsentry/raven-go"
```

Comment:
I am not sure what this is about...

Reply:
Yeah, I did not look at that. Sorry.

Reply:
I guess this is the reason the dep on `raven-go` stayed in `go.mod`...